### ISSUE TITLE
Model selector not responding

### ISSUE THREAD

**Issue by reginacas** [reginacas/sense-ui#147]
Reported by @alekunkel : 

> Status: UI Unresponsive / Cannot switch model.
> 
> Description:
> I am unable to change the AI model in the frontend interface. The selector does not allow me to switch between models at all (it seems stuck or unresponsive). Therefore, I could not test different models.
